RELEASES.md

```diff
@@ -677,3 +677,14 @@
 10.4.4 2020-01-30
 -----------------
  - fix: client login failure leak , cherry-picked from version 11
+
+Deprecated versions
```

Comment:
Please change this to version headings, with the changelist simply saying deprecated.

```
10.4.6 2020-03-05
-------------------
- Deprecated

Reply:
Done.